docs/configuration.md

```diff
@@ -1752,6 +1752,15 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing R shell in both client mode and cluster mode. For now sparkr shell only supports
```

Comment:
It might be not so proper to mention "executing R shell in both client mode and cluster mode" in Spark's doc, AFAIK it is a Livy-only feature. sparkr can only run in client mode for now in Spark. So described here may confuse the user.